includes/class-wc-stripe-payment-tokens.php

```diff
@@ -82,6 +82,50 @@ public static function customer_has_saved_methods( $customer_id ) {
 		return $has_token;
 	}
 
+	/**
+	 * Creates and add a token to an user, based on the payment_method object
+	 *
+	 * @param   stdClass                                        $payment_method Stripe payment method instance to be added.
+	 * @param   WC_Stripe_Customer                              $user User to attach payment method to.
+	 * @return  WC_Payment_Token_CC|WC_Payment_Token_SEPA The WC object for the payment token.
+	 */
+	public function add_token_to_user( $payment_method, $user ) {
+		// Clear cached payment methods.
+		$user->clear_cache();
+
+		// TODO: change 'sepa_debit' to the SEPA reference in the class that will be created to match Stripe's value
```

Comment:
In case #1777 gets merged before this PR we can make use of the new class and complete this todo already. Otherwise that can be done in other PR.